pkg/avx/collector.go

```diff
@@ -38,6 +40,8 @@ var (
 			"cgroup",
 		}, nil,
 	)
+
+	BpfBinaryName = "avx512.o"
```

Comment:
This (exported var) should have a docment to make go lint shut up.

Reply:
fixed.